themes/all.ini

```diff
@@ -16,11 +16,11 @@ catppuccin/latte/comment = ('#8c8fa1', False, True)
 catppuccin/latte/string = ('#40a02b', False, False)
 catppuccin/latte/number = ('#fe640b', False, False)
 catppuccin/latte/instance = ('#d20f39', False, True)
-catppuccin/frappe/name = Catppuccin Frappe
+catppuccin/frappe/name = Catppuccin Frappé
```

Comment:
Can revert if this isn't supported.

Reply:
I really don't want to install Spyder to test this lol, I'd prefer the revert if that's alright.